docker/README.md

```diff
@@ -0,0 +1,61 @@
+# VRX Docker support
```

Comment:
How does this file play with the [Docker installation instructions from the wiki](https://github.com/osrf/vrx/wiki/docker_install_tutorial)?

Reply:
Prerequisites are the same. No need to clone dockwater repository anymore, just run the `docker compose` command.